dev/bin/manager-run.sh

```diff
@@ -5,12 +5,13 @@
 #
 # Env vars:
 # - MANAGED_CONNECTORS_CLUSTER_ID: cluster where managed connectors will be deployed (required only if MC actions are used, default="empty")
-# - MANAGED_KAFKA_INSTANCE_NAME: set the managed kafka instance name (required)
```

Comment:
`manager-run.sh` refers to `kafka-setup.sh` in comments, could we please revisit it?

Reply:
done